pkg/apis/experimental/install/install.go

```diff
@@ -51,14 +51,16 @@ func init() {
 		GroupVersion: groupVersion,
```

Comment:
The line above that says "registeredGroupVersions[0]" can't possibly be right, can it?